src/sdl2_compat.c

```diff
@@ -343,8 +343,10 @@ static QuirkEntryType quirks[] = {
 
 #ifdef __linux__
 static void OS_GetExeName(char *buf, const unsigned maxpath) {
+    int ret;
     buf[0] = '\0';
-    readlink("/proc/self/exe", buf, maxpath);
+    ret = readlink("/proc/self/exe", buf, maxpath);
+    (void)ret;
```

Comment:
Does it still warn if you just _`(void)`_ the readlink call, instead? E.g. like:
```diff
--- a/src/sdl2_compat.c
+++ b/src/sdl2_compat.c
@@ -345,5 +345,5 @@
 static void OS_GetExeName(char *buf, const unsigned maxpath) {
     buf[0] = '\0';
-    readlink("/proc/self/exe", buf, maxpath);
+    (void) readlink("/proc/self/exe", buf, maxpath);
 }
 #elif defined(_WIN32)
```

Reply:
yes, sorry I wasn't clear:

nothing:
```
sdl2_compat.c:347:5: error: ignoring return value of ‘readlink’ declared with attribute ‘warn_unused_result’ [-Werror=unused-result]
  347 |     readlink("/proc/self/exe", buf, maxpath);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```


with (void)readlink:
```
sdl2_compat.c:347:11: error: ignoring return value of ‘readlink’ declared with attribute ‘warn_unused_result’ [-Werror=unused-result]
  347 |     (void)readlink("/proc/self/exe", buf, maxpath);
      |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Jeez, that's overzealous..

OK then, resolving this comment